### DISCUSSION TITLE
Can the Panel component retain elements when collapsed?

### DISCUSSION THREAD

**Discussion by ryrocks** [primefaces/community#347] [PrimeReact] [answered]
I wonder if the **Panel** component offers an API similar to **TabView**'s `renderActiveOnly`. 
Specifically, I'm looking for a way to render content elements even when the panel is collapsed

Official example:
https://codesandbox.io/s/primereact-demo-forked-jly9p8

To provide context, I'm using React-hook-form. If no elements are rendered, it would not trigger validation.

**Comment by gucal** [comment, marked as answer]
Hi,
The panel component does not have a working prop like renderActiveOnly.
It can be added as a new feature. Thank you for the suggestion.

**Comment by mertsincan** [comment]
Hi,
We can also expect upvotes from other users. We will be happy to implement it when we get enough upvotes.

**Reply by elliotspall** [reply]
Would love to see this feature added! Having the same issues here also using hook form...

**Comment by XYIAN** [comment]
I would really like this feature to be added! This is an issue I had to create many workarounds for as well as helped a lot of other users solve this issue on stackoverflow - having a simple prop for this would save a lot of time! A lot of other UI frameworks have this as default such as ext or MUI so adding this feature would help PR be more competitive!
